CONTRIBUTING.md

```diff
@@ -0,0 +1,57 @@
+# Contributing
+
+`pwru` is an open source project. The userspace code is licensed under [Apache-2.0](LICENSE), while the BPF under [BSD 2-Clause](bpf/LICENSE.BSD-2-Clause) and [GPL-2.0](bpf/LICENSE.GPL-2.0). Everybody is welcome to contribute. Contributors are required to follow the [CNCF Code of
+Conduct](https://github.com/cncf/foundation/blob/main/code-of-conduct.md) and must adhere to the [Developer Certificate of Origin](https://developercertificate.org/) by adding a Signed-off-by line to their commit messages.
+
+## Getting Started
+
+- Read the [README](https://github.com/cilium/pwru#readme) to understand how `pwru` works and its use cases.
+- Check out the [issues list](https://github.com/cilium/pwru/issues) for open tasks, especially those labeled `good first issue`.
+
+## Development Guidelines
+
+- Follow Cilium's general [Developer Guide](https://docs.cilium.io/en/stable/contributing/development/) for coding standards and best practices.
+- Before submitting a pull request, review the [Code Review Guidelines](https://docs.cilium.io/en/stable/contributing/code-reviews/).
```

Comment:
Broken link, did you mean https://docs.cilium.io/en/stable/contributing/development/contributing_guide/#getting-a-pull-request-merged ?